lib/proxi/local/proxi_datasets.py

```
@@ -764,24 +733,29 @@ def rebuild_extended_data_cache(self):
             identifier = row[0]
             announce_date = row[9]   # FIXME
 
-            previous_extended_data_date ='2026-01-01'
+            previous_extended_data_date ='2026-05-01'
             #if identifier == 'PXD070494':
             if announce_date >= previous_extended_data_date:
                 print(f"irow={irow}  identifier={identifier}, announce_date={announce_date}")
                 status_code, message, dataset = self.get_dataset(identifier)
```

Ruff (0.15.12)

[warning] Unpacked variable `status_code` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)